contracts/lib/Transfer.sol

```diff
@@ -50,19 +50,19 @@ library Transfer {
 
   /// @notice Verifies whether or not a `Transfer.Transaction` meets the terms set by a
   /// `Transfer.Terms` object based on the limit information of how much can be transferred
-  /// @param tx A `Transfer.Transaction` struct
+  /// @param txn A `Transfer.Transaction` struct
```

Comment:
This also appears in https://github.com/counterfactual/contracts/blob/87397d6cd8ffbcb468c78c92d83d51d8c0f09daf/contracts/delegateTargets/ConditionalTransfer.sol#L21